setup.py

```diff
@@ -55,10 +55,13 @@
 
 version = imp.load_source('pyspectral.version', 'pyspectral/version.py')
 
+description = ('Reading and manipulaing satellite sensor spectral responses and the ' +
+               'solar spectrum, to perfom various corrections to VIS and NIR band data')
+
 setup(name='pyspectral',
       version=versioneer.get_version(),
       cmdclass=versioneer.get_cmdclass(),
-      description='Reading and manipulaing satellite sensor spectral responses and the solar spectrum, to perfom various corrections to VIS and NIR band data',
+      description=description,
```

Comment:
This complicates the PR, but you could also add a `long_description` with the content of the README so it shows up on PyPI.

Reply:
It was there already :-)
It is maybe just confusing that the `description` is quite long!

Reply:
Ah yes, just missed it because `long_description` wasn't next to `description` and github didn't default to showing me all of the file.

Reply:
Fine, I moved the description up to the other one, thanks!